PlasterTemplate/PlasterTemplates/GitLab/psake.ps1

```diff
@@ -235,13 +238,15 @@ Task Deploy -Depends TestAfterBuild {
         Write-Error "PowerShell repository API key not found"
     }
 
-    <#
-    "`tTesting for GitLab Personal Access Token"
-    if (!$ENV:GitLab_PAT) {
-        Write-Error "GitLab personal access token not found"
-    }
-    #>
 
+<%
+    if ($PLASTER_PARAM_DeployDocs -eq 'Yes') {
+        '   "`tTesting for GitLab Personal Access Token"'
+        '   if (!$ENV:GitLab_PAT) {'
+        '       Write-Error "GitLab personal access token not found"'
+        '   }'
```

Comment:
These need one more leading space inside the quotes as it's only 3 at the moment